spec/handlers/c/method_handler_spec.rb

```diff
@@ -213,6 +213,7 @@
     foo = Registry.at('Foo#foo?')
     expect(foo.docstring).to eq 'DOCSTRING'
     expect(foo.tag(:return).types).to eq ['Boolean']
+    expect(foo.tags(:return).size).to eq 1
```

Comment:
Without the fix to `handler_methods.rb` this test still passed. And I don't quite understand why. I would have expected this to yield 2 and fail when YARD added an extra `@return` tag.

Reply:
This is probably because one of the tags ends up in your overload. You probably want an implementation more normalized with the Ruby handler:

https://github.com/lsegal/yard/blob/master/lib/yard/handlers/ruby/method_handler.rb#L41-L49

Reply:
There are no extra tags on the method defined in this specific test, which would explain why the result hasn't changed.